compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -545,15 +560,18 @@ struct __sanitizer_dirent64 {
 extern unsigned struct_sock_fprog_sz;
 #endif
 
-#if SANITIZER_HAIKU
+#  if SANITIZER_HAIKU
+typedef int __sanitizer_clock_t;
+#  elif SANITIZER_AIX
 typedef int __sanitizer_clock_t;
-#elif defined(__x86_64__) && !defined(_LP64)
+typedef int __sanitizer_clockid_t;
```

Comment:
Move `__sanitizer_clockid_t` definition to the next block where it is defined for other platforms.